Library/Backends/ZendEngine3/Backend.php

```diff
@@ -362,7 +362,7 @@ public function assignZval(Variable $variable, $code, CompilationContext $contex
         }
     }
 
-    public function returnString($value, CompilationContext $context, $useCodePrinter = true)
+    public function returnString($value, CompilationContext $context, $useCodePrinter = true, $doCopy = true)
```

Comment:
Yes because ZendEngine3 extends ZendEngine2 where returnString is with additional parameters